bin/update-release-version.php

```diff
@@ -140,7 +140,8 @@ function get_next_minor_version(array $versions): array
     ];
 }
 
-if ($argc !== 2) {
+// Allow 2 arguments as the bump-version action always passes a version number, even when not needed
+if (! in_array($argc, [2, 3])) {
```

Comment:
The version bump script always passes the version number as an argument, so I decided to allow an additional argument without using it.